### PR TITLE
Add support for float and double pixel types

### PR DESCRIPTION
**Pull request by chris-allan**
.

### CODE REVIEW

**Comment by joshmoore** [comment]
Ran successfully against Image:1884807 (idr0021). Still needs visual confirmation.

File: src/main/java/com/glencoesoftware/bioformats2raw/Converter.java [hunk 1]
```diff
@@ -657,23 +659,37 @@ private void processTile(
...
+        bb = bb.order(ByteOrder.LITTLE_ENDIAN);
+      }
+      switch (pixelType) {
+        case FormatTools.INT8:
+        case FormatTools.UINT8: {
           dataBlock = new ByteArrayDataBlock(size, gridPosition, tile);
           break;
-        case 2:
+        }
+        case FormatTools.INT16: {
```
Comment (melissalinkert) [inline src/main/java/com/glencoesoftware/bioformats2raw/Converter.java:672]: Add ```FormatTools.UINT16``` too?

Comment (joshmoore) [reply src/main/java/com/glencoesoftware/bioformats2raw/Converter.java]: Ha! Was just about to say I had run into this: ```loci.formats.FormatException: Unsupported pixel type: uint16```. Thanks, guys.

**Comment by chris-allan** [comment]
@joshmoore: You want final veto here based on visual review or are you happy for this to go in?

**Pull request merged**